### PR TITLE
python3Packages.batchgenerators: init at 0.19.7

### PR DESCRIPTION
**Pull request by bcdarwin**
python3Packages.threadpoolctl: init at 2.0.0

###### Motivation for this change

Add a package.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [NA] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [NA] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [NA] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/threadpoolctl/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  format = "flit";
+
+  src = fetchFromGitHub {
+    owner = "joblib";
+    repo = pname;
+    rev = version;
+    sha256 = "16z4n82f004i4l1jw6qrzazda1m6v2yjnpqlp71ipa8mzy9kw7dw";
+  };
+
+  nativeBuildInputs = [ flit ];
```
Comment (jonringer) [inline pkgs/development/python-modules/threadpoolctl/default.nix]: `format = "flit";` brings this in for you
```suggestion
```

**Comment by bcdarwin** [comment]
@jonringer -- resolved, thanks.

**Review by jonringer — APPROVED** [review]
diff LGTM
commits LGTM
has tests :+1: 

```
https://github.com/NixOS/nixpkgs/pull/81739
4 package built:
python37Packages.batchgenerators python37Packages.threadpoolctl python38Packages.batchgenerators python38Packages.threadpoolctl
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build python37Packages.batchgenerators python37Packages.threadpoolctl python38Packages.batchgenerators python38Packages.threadpoolctl

**Comment by bcdarwin** [comment]
`python38Packages.numexpr` is timing on out Darwin.

**Pull request merged**